pkg/plugins/accesscontrol.go

```diff
@@ -3,17 +3,29 @@ package plugins
 import (
 	"github.com/grafana/grafana/pkg/models"
 	ac "github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/setting"
 )
 
 const (
+	// Plugins actions
+	ActionInstall       = "plugins:install"
+	ActionSettingsWrite = "plugins.settings:write"
```

Comment:
Maybe we should rename this one to `plugins:write`. That action would allow users to update settings and enable / disable plugin